### PR TITLE
Adding doc template to function invPolyArcLength3

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
An input and an output are still to be determined

### CODE REVIEW

File: src/Functions/invPolyArcLength3.m [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
-% mail@andreashusch.de, husch.andreas@chl.lu
-function t = invPolyArcLength3(polyCoeff, arcLength)
-t = zeros(size(arcLength)); % default t=0 (i.e. in caes arcLength is 0)
+% USAGE:
+%
+%    t = invPolyArcLength3(polyCoeff, arcLength)
+%
+% INPUTS: 
+%    polyCoeff:     Coefficient matrix
+%    arcLength:
```
Comment (Daniel-Tojal) [inline src/Functions/invPolyArcLength3.m]: @adhusch Could you please provide a description of the inputs and outputs?

Comment (adhusch) [reply src/Functions/invPolyArcLength3.m]: arcLength: arc length in mm (as stated above)

**Review by adhusch — CHANGES_REQUESTED** [review]

File: src/Functions/invPolyArcLength3.m (hunk 1 above)
Comment (Daniel-Tojal) [reply src/Functions/invPolyArcLength3.m]: Oh right, thank you very much.

**Comment by laurentheirendt** [comment]
Thanks @Daniel-Tojal 👍

**Pull request merged**